src/components/DayPickerRangeController.jsx

```diff
@@ -910,7 +911,9 @@ export default class DayPickerRangeController extends React.PureComponent {
     });
 
     const prevMonth = currentMonth.clone().subtract(2, 'months');
-    const prevMonthVisibleDays = getVisibleDays(prevMonth, 1, enableOutsideDays, true);
+    const prevMonthVisibleDays = getVisibleDays(
+      prevMonth, 1, enableOutsideDays, firstDayOfWeek, true,
```

Comment:
```suggestion
      prevMonth,
       1,
       enableOutsideDays,
       firstDayOfWeek,
       true,
```
(please don't run prettier in your editor directly; run eslint autofix instead, and if the repo is configured to use prettier, then it will be invoked)